project1/import.py

```diff
@@ -0,0 +1,29 @@
+import os
+import csv
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
+if not os.getenv("DATABASE_URL"):
+    raise RuntimeError("DATABASE_URL is not set")
+
+engine= create_engine(os.getenv("DATABASE_URL"))
+db =scoped_session(sessionmaker(bind=engine))
+
+def main():
+    # db.execute("CREATE TABLE users (id SERIAL PRIMARY KEY, username VARCHAR NOT NULL, password VARCHAR NOT NULL)")
+    # db.execute("CREATE TABLE reviews (isbn VARCHAR NOT NULL,review VARCHAR NOT NULL, rating INTEGER NOT NULL,username VARCHAR NOT NULL)")
+    db.execute("CREATE TABLE books (isbn VARCHAR PRIMARY KEY,title VARCHAR NOT NULL,author VARCHAR NOT NULL,year INTEGER NOT NULL)")
+    f=open("books.csv")
+    reader =csv.reader(f)
+    for isbn,title,author,year in reader:
+        if year == "year":
+            print('skipped first line')
+        else:    
```

Comment:
You can use : `next(reader)` to skip first line in csv instead of if-else condition

Reply:
Yep!!! Got it. Thanks for the help.